dpnp/dpnp_algo/dpnp_algo.pxd

```diff
@@ -87,7 +86,6 @@ cdef extern from "dpnp_iface_fptr.hpp" namespace "DPNPFuncName":  # need this na
         DPNP_FN_ERF
         DPNP_FN_ERF_EXT
         DPNP_FN_EYE
-        DPNP_FN_EYE_EXT
```

Comment:
I think you can also remove `DPNP_FN_EYE`, `DPNP_FN_FLATTEN`, `DPNP_FN_REPEAT`, and `DPNP_FN_REPEAT` in this file.